ASV44/sample-ios-app
# Add basic setup of web socket chat

### Background

As part of networking example is implemented connection to websocket server and is implemented basic chat communication which supports basic message sending and receiving. Chat module represents client part of [Chat Message Broker](https://github.com/ASV44/ChatMessageBroker) which provides basic chat features. 

### What has been done?
1. Add basic websocket connection and message handling
2. Add simple UI of message input and message display.

### How to test?
1. Clone [Chat Message Broker](https://github.com/ASV44/ChatMessageBroker) and follow all instruction from `README` for running instance of broker app
2. Open app and click on `Chat Example`
3. As first message input user name used for chat
4. Ensure from server logs that new user with provided user name is connected.
5. Run one more app instance or CLI chat client and register one more user in workspace
6. Send messages between users.

### Checklist
